src/app/usersList/UsersList.tsx

```diff
@@ -0,0 +1,76 @@
+import { Fragment } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+import { CodeBlock } from 'ui/codeBlock/CodeBlock';
+import { useQuery } from 'hooks/useQuery/useQuery';
+
+type SortType = 'asc' | 'desc';
+
+export const UsersList = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const sort = searchParams.get('sort') as SortType;
+  const page = searchParams.get('page');
+
+  const { data: usersResponse, isFetched: areUsersFetched } = useQuery(
+    'getUsersList',
+    { page: page || undefined },
```

Comment:
should we use `??` instead of `||`?

Reply:
it doesn't matter here. `page` is `string` lub `null`, query needs `string | undefined` so
no matter what will be in `page` when I use `||` lub `??` result will be the same